src/mumble/JackAudio.cpp

```diff
@@ -1059,6 +1125,84 @@ bool JackAudioOutput::process(const jack_nframes_t frames) {
 	return true;
 }
 
+void JackAudioOutput::prepareOutputBuffers(unsigned int frameCount, QList< AudioOutputBuffer * > &qlMix,
```

Comment:
Maybe this implementation should contain a short-circuit code that delegates to the default base-class implementation if the multichannel recording is not active